pom.xml

```diff
@@ -169,6 +170,12 @@
       <artifactId>vertx-web-api-contract</artifactId>
       <version>${vertx.version}</version>
     </dependency>
+    <!-- Allows us to suppress warnings from the AWS SDK (see https://github.com/aws/aws-sdk-java/issues/1707) -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
+      <version>${jcl.over.slf4j.version}</version>
```

Comment:
I think we should add a `<scope>test</scope>` so that the dependency won't be packaged into the final application jar. It's useful in our test environment, but I don't think we need it in the runtime? Also, with the next version of vertx-super-s3 the AWS dependency will also be removed (i.e., marked as a test dependency only).

Reply:
When we put the scope on it we could move the dependency below the ` <!-- Below dependencies only used for testing -->` line in the pom.xml too.